src/content/structured/get-started/react.mdx

```diff
@@ -103,3 +103,38 @@ module.exports = withTM(nextConfig);
 The `@ukic/react` package will need to be transformed before you can use these components in Jest tests.
 
 Add a `transformIgnorePatterns` field with the value `["/node_modules/(?!@ukic/react)"]` to your Jest config.
+
+## Working with slotted SVGs
+
+When wanting to slot an SVG into one of the components, you should import SlottedSVG in your React files to prevent an error stating that property 'slot' doesn't exist.
+
+```jsx
+import { SlottedSVG } from "@ukic/react";
+```
+
+<br />
+
+Then update your code to replace any instances of `<svg>` with `<SlottedSVG>`.
+
+As an example, when using IcTopNavigation, if your existing code looks like this:
+
+```jsx
+<IcTopNavigation appTitle="App title">
+    <svg
+        slot="app-icon"
+        ...
+    >
+        <path ... />
```

Comment:
I think the path would also be a prop on the svg element here (like you changed for the SlottedSVG), rather than a child\inner content